src/screens/editor/container/editorContainer.tsx

```diff
@@ -111,7 +111,15 @@ class EditorContainer extends Component<any, any> {
             thumbIndex: draftThumbIndex,
           })
         }
-
+        // load beneficiaries and rewards data from meta field of draft
+        if(_draft.meta && _draft.meta.rewardType){
```

Comment:
@aliseyalvi  this placement of draft options loading might not handle draft loading from floating button. or perhaps you handled it somewhere else...?

<img width="276" alt="Screenshot 2022-06-24 at 1 53 31 AM" src="https://user-images.githubusercontent.com/6298342/175397869-09bc7fa5-a46d-4593-ba36-bbe560221b90.png">

Reply:
@noumantahir  fixed here https://github.com/ecency/ecency-mobile/pull/2356/commits/a10232977e456e593a40b65bc078203d1e051cae